### ISSUE TITLE
EGL libs missing

### ISSUE THREAD

**Issue by theblixguy** [Fevax/hero2ltexx-logcats#1]
Noticed in your latest logcat `Abort message: 'couldn't find an OpenGL ES implementation'`

Can you confirm the EGL libs are present in /system/lib/egl or /vendor/lib/egl?

**Comment by Fevax** [comment]
06-23 00:21:57.915  2904  2904 E libEGL  : load_driver(/vendor/lib64/egl/libGLES_mali.so): dlopen failed: library "libion.so" not found
06-23 00:21:57.915  2904  2904 F libEGL  : couldn't find an OpenGL ES implementation

This?
changing some code i forgot to include libion.so and thats why i have the error

**Comment by theblixguy** [comment]
No, it was in a different logcat. Anyhow, you need the EGL files in the folders I mentioned or else openGL will not work
